integration/integrate-with-qradar.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="integrate-with-qradar"]
+= Integrating with QRadar
+include::modules/common-attributes.adoc[]
+:context: integrate-with-qradar
+
+toc::[]
+
+[role="_abstract"]
+You can configure {product-title-short} to send events to QRadar by configuring a generic webhook integration in {product-title-short}.
+
+The following steps represent a high-level workflow for integrating {product-title-short} with QRadar:
+
+. In {product-title-short}:
+.. Configure the generic webhook. 
++
+[NOTE]
+====
+When configuring the integration in {product-title-short}, in the *Endpoint* field, use the following example as a guide: `<URL to QRadar Box:<Port of Integration>`.
```

Comment:
```suggestion
When configuring the integration in {product-title-short}, in the *Endpoint* field, use the following example as a guide: `<URL to QRadar Box>:<Port of Integration>`.
```

Is there a missing angle bracket?

Reply:
That's exactly how they have it in the doc, so I don't know. I can check.

Reply:
Confirmed from our ACS team that it needs to be URL:port (or an endpoint) so yes, good catch, needs another bracket!